javascript/ql/src/Security/CWE-094/CodeInjection.qhelp

```diff
@@ -40,5 +57,13 @@ OWASP:
 <li>
 Wikipedia: <a href="https://en.wikipedia.org/wiki/Code_injection">Code Injection</a>.
 </li>
+<li>
+OWASP:
+<a href="https://owasp.org/www-project-web-security-testing-guide/latest/4-Web_Application_Security_Testing/07-Input_Validation_Testing/18-Testing_for_Server_Side_Template_Injection">Server Side Template Injection</a>.
```

Comment:
This link 404's for me. Could you check that the URL is correct?

Reply:
Removed the link. Whatever it was pointing to I can't find it now.

Reply:
@asgerf were you looking for this:
https://owasp.org/www-project-web-security-testing-guide/stable/4-Web_Application_Security_Testing/07-Input_Validation_Testing/18-Testing_for_Server-side_Template_Injection.html
?